nf_core/download.py

```diff
@@ -960,6 +978,70 @@ def prioritize_direct_download(self, container_list):
                 d[k] = c
         return sorted(list(d.values()))
 
+    def gather_registries(self, workflow_directory):
```

Comment:
```suggestion
    def gather_registries(self, workflow_directory:str) -> None:
```

Reply:
we are starting to use typehinting everywhere, so better not to add any more functions without 🙂